docs/guide.md

```diff
@@ -45,6 +45,10 @@ For a good experience, we recommend at least:
 
 You can use whatever linux distribution floats your boat but in this guide we assume Debian on Google Cloud.
 
+In order to work properly, your enviornment should have the following enabled:
+
+- WebSockets (used to communicate between code-server client and server)
```

Comment:
Since there's one requirement, it reads a bit weird being a list. Something like:

```
WebSocket's are required to use code-server.
```

Reply:
I guess the rationale for a list is that we're probably going to add more here later?

Perhaps:

```suggestion
* WebSockets, which code-server uses to communicate between the browser and server
```

Reply:
Yeah, I'll change to one line instead of a list though!